src/iden3comm/constants.ts

```diff
@@ -38,5 +37,73 @@ export const PROTOCOL_MESSAGE_TYPE = Object.freeze({
  */
 export enum MediaType {
   ZKPMessage = 'application/iden3-zkp-json',
-  PlainMessage = 'application/iden3comm-plain-json'
+  PlainMessage = 'application/iden3comm-plain-json',
+  SignedMessage = 'application/iden3comm-signed+json'
 }
+
+/**
+ * Universal resolver url
+ */
+export const UNIVERSAL_RESOLVER_URL = 'https://dev.uniresolver.io/1.0/identifiers';
+
+export const SUPPORTED_PUBLIC_KEY_TYPES = {
+  ES256: ['JsonWebKey2020'],
+  ES256K: [
+    'EcdsaSecp256k1VerificationKey2019',
```

Comment:
we need to handle ethereumAddress (if ethereumAddress - > also use custome singer)

Reply:
This map is about supported public key types, handling of ethereumAddress in jws packer

Reply:
This map is about supported public key types, handling of ethereumAddress in jws packer